### ISSUE TITLE
check-property-names reports false positives for duplicate property names when multiple typedefs within one comment

### ISSUE THREAD

**Issue by Standard8** [gajus/eslint-plugin-jsdoc#1619]
## Expected behavior

No errors should be reported due to using two typedefs in a single comment. As far as I can tell, TypeScript allows this and works fine.

## Actual behavior

An error is reported: `error  Duplicate @property "type"  jsdoc/check-property-names`.

## ESLint Config

```js
import jsdoc from "eslint-plugin-jsdoc";

export default [{
  plugins: {jsdoc},
    rules: {
      "jsdoc/check-property-names": "error",
    }
}]
```

## ESLint sample

```js
/**
 * @typedef {object} foo
 * @property {string} type
 *
 * @typedef {object} bar
 * @property {number} type
 */
```

## Environment

- Node version: v25.2.1
- ESLint version v9.39.2
- `eslint-plugin-jsdoc` version: 61.5.0

**Comment by brettz9** [comment]
This has since been released. The current version is v62.0.0.
